datahub-web-react/src/app/ingest/source/builder/RecipeForm/SecretField/SecretField.tsx

```diff
@@ -36,25 +36,30 @@ interface SecretFieldProps {
 
 function SecretField({ field, secrets, removeMargin, refetchSecrets }: SecretFieldProps) {
     return (
-        <StyledFormItem name={field.name} label={field.label} tooltip={field.tooltip} removeMargin={!!removeMargin}>
-            <Select
-                showSearch
-                filterOption={(input, option) => !!option?.children.toLowerCase().includes(input.toLowerCase())}
-                dropdownRender={(menu) => (
-                    <>
-                        {menu}
-                        <StyledDivider />
-                        <CreateSecretButton refetchSecrets={refetchSecrets} />
-                    </>
-                )}
-            >
-                {secrets.map((secret) => (
-                    <Select.Option key={secret.urn} value={`\${${secret.name}}`}>
-                        {secret.name}
-                    </Select.Option>
-                ))}
-            </Select>
-        </StyledFormItem>
+        <>
+            <StyledFormItem name={field.name} label={field.label} tooltip={field.tooltip} removeMargin={!!removeMargin}>
+                <Tooltip title="This field requires you to use a DataHub Secret. For more information on Secrets in DataHub, please review the docs.">
```

Comment:
can you make "the docs" a link to https://datahubproject.io/docs/ui-ingestion/#creating-a-secret

Reply:
Fixed and Committed!